cli/tests/integrations/test_task.py

```diff
@@ -257,6 +257,30 @@ def test_ls_bad_path():
         returncode=1)
 
 
+def test_ls_completed():
+    # create a completed task
+    with app(SLEEP_COMPLETED, 'test-app-completed'):
+        # get its task id
+        task_id_completed = _get_completed_task_id()
+
```

Comment:
It would lead to other tests (`*_completed`) that rely upon that to fail. Unless you move this test before the other tests. I do not like the idea of tests being order sensitive